jenkins/jjb/templates/scenario-tests.yaml

```diff
@@ -0,0 +1,149 @@
+              def summary = manager.createSummary("gear2.gif")
+              summary.appendText("Infrastructure Builds:<ul>", false)
+              summary.appendText("<li><b>vSphere Version</b> - ${{vsphere_version}}</li>", false)
+              summary.appendText("<li><b>vCenter</b> - ${{vc_build}}</li>", false)
+              summary.appendText("<li><b>ESXi</b> - ${{esx_build}}</li>", false)
+      - email-ext:
+          recipients: vic-eng@vmware.com
+          reply-to: vic-eng@vmware.com
+          success: true
+          aborted: true
+          failure: true
+          content-type: html
+          send-to:
+            - recipients
+          subject: 'vmware/{repo} Scenario Testing - ${{VSPHERE_VERSION}}/{repo}/{branch} - Build: $BUILD_NUM'
+          body: |
+              Download link for the test logs is in the <a href="${{BUILD_URL}}/console">build console output</a>
+              <p>
+              ${{VSPHERE_VERSION}} Jenkins Nimbus Scenario Job: <a href="${{BUILD_URL}}">${{BUILD_URL}}</a>
+              <p>
+              Run Report: ${{ROBOT_REPORTLINK}}
+              <p>
+              ${{FILE,path="{repo}/console.log"}}
+      - slack:
+          room: '#notifications'
+          auth-token-credential-id: 'SlackIntegrationToken'
+          notify-success: True
+          notify-aborted: True
+          notify-failure: True
+          base-url: 'https://vmware-vic.slack.com/services/hooks/jenkins-ci/'
+          include-custom-message: True
+          custom-message: 'vmware/{repo} Scenario Testing - ${{VSPHERE_VERSION}}/{repo}/{branch} - $BUILD_TIMESTAMP'
+
+- job-template:
+    name: 'vic-{version}-scenario'
+    defaults: 'scenario-tests'
+    builders:
+      - shell: |
+          #!/bin/bash
+          set -x
+          dest="vic-ci-logs/vmware/{branch}/"
+
+          cat > build.envfile <<ENVS
+          PARALLEL_JOBS={parallel_jobs}
+          NIMBUS_RETRY_ATTEMPTS=5
+          NIMBUS_RETRY_DELAY=1m
+          LOG_UPLOAD_DEST=$dest
+          GIT_COMMIT=$GIT_COMMIT
+          BUILD_TIMESTAMP=$BUILD_TIMESTAMP
+          BUILD_ID=$BUILD_ID
+          ESX_BUILD=$ESX_BUILD
+          VC_BUILD=$VC_BUILD
+          BUILD_TAG={repo}_${{BUILD_NUM}}
+          VCH_BUILD=${{BUILD_NUM}}
+          VCH_BRANCH={branch}
+          ENVS
+
+          docker run --rm -v "${{WORKSPACE}}":/go --env-file vic-internal/{repo}-scenario-{version}-secrets.list --env-file build.envfile --name=${{BUILD_TAG}} gcr.io/eminent-nation-87317/vic-integration-test:1.48 vic-tools/jenkins/jobs/vic-scenario/build.sh ${{VSPHERE_VERSION}} ${{TEST_CASES}}
+
+- job-template:
+    name: 'vic-product-{version}-scenario'
+    defaults: 'scenario-tests'
+    builders:
+      - shell: |
+          #!/bin/bash
+          set -x
+
+          ./vic-tools/jenkins/jobs/vic-product-scenario/selenium_grid.sh create {version}
+
+          cat > build.envfile <<ENVS
+          PARALLEL_JOBS={parallel_jobs}
+          ROBOT_REPORT=report
```

Comment:
I don't see this used anywhere

Reply:
This is used in vic-tools/jenkins/jobs/vic-scenario/build.sh